tests/integration/common/data/interfaces/test_user.py

```diff
@@ -326,3 +335,40 @@ def test_remove_all_roles_from_user(self, db_session, factories) -> None:
         interfaces.user.remove_all_roles_from_user(user)
         assert db_session.scalar(select(func.count()).select_from(UserRole)) == 0
         assert user.roles == []
+
+
+class TestInvitations:
+    @pytest.mark.freeze_time("2023-10-01 12:00:00")
+    def test_create_invitation(self, db_session, factories):
+        invitation = interfaces.user.create_invitation(email="test@email.com", role=RoleEnum.MEMBER)
+        invite_from_db = db_session.get(Invitation, invitation.id)
+        assert invite_from_db is not None
+        assert invite_from_db.email == "test@email.com"
+        assert invite_from_db.role == RoleEnum.MEMBER
+        assert invite_from_db.expires_at_utc == datetime.strptime("2023-10-08 12:00:00", "%Y-%m-%d %H:%M:%S")
```

Comment:
```suggestion
        assert invite_from_db.expires_at_utc == datetime(2023, 10, 8, 12, 0, 0)
```

nit: not sure the extra parsing from a string is useful here